src/main/java/com/cloudblue/connect/api/models/CBCUser.java

```diff
@@ -17,6 +17,9 @@
     @JsonProperty
     private String name;
 
+    @JsonProperty
+    private String externalId;
```

Comment:
'User' object does not have external id property. 

https://connect.cloudblue.com/community/api/openapi/#operations-Accounts-account_users_list_listAccountUsers